tests/common.go

```diff
@@ -4107,11 +4152,24 @@ func AddLabelToResource(spec interface{}, key string, val string) error {
 	return fmt.Errorf("spec is of unknown resource type")
 }
 
+// GetClusterConfigPath returns kubeconfig for given cluster
+func GetClusterConfigPath(clusterConfig string) (string, error) {
+	kubeconfigs := os.Getenv("KUBECONFIGS")
+	kubeconfigList := strings.Split(kubeconfigs, ",")
+	for _, kubeconfig := range kubeconfigList {
+		if kubeconfig == clusterConfig {
+			log.Infof("Cluster config path: %s", fmt.Sprintf("%s/%s", KubeconfigDirectory, kubeconfig))
+			return fmt.Sprintf("%s/%s", KubeconfigDirectory, kubeconfig), nil
+		}
+	}
+	return "", nil
+}
+
 // GetSourceClusterConfigPath returns kubeconfig for source
 func GetSourceClusterConfigPath() (string, error) {
 	kubeconfigs := os.Getenv("KUBECONFIGS")
 	if kubeconfigs == "" {
-		return "", fmt.Errorf("Failed to get source config path. Empty KUBECONFIGS environment variable")
+		return "", fmt.Errorf("failed to get source config path. Empty KUBECONFIGS environment variable")
```

Comment:
How about using "failed" throughout the function? Or should we leave it as it is?

Reply:
Will take this up as part of my next PR  where I will make changes for all testcases: 
https://portworx.atlassian.net/browse/PA-1337
https://portworx.atlassian.net/browse/PA-1336